packages/opentelemetry-core/package.json

```diff
@@ -10,7 +10,7 @@
   "types": "build/src/index.d.ts",
   "repository": "open-telemetry/opentelemetry-js",
   "scripts": {
-    "test": "nyc ts-mocha -p tsconfig.json test/**/*.test.ts",
+    "test": "nyc ts-mocha -p tsconfig.json test/**/*.test.ts --exclude 'test/platform/browser/**/*.ts'",
```

Comment:
Just curious to know, why do we have to do this? Also, do you think we should add this to other common packages like api, tracing etc.

Reply:
if there are only dedicated tests for browser then this is the way to avoid testing them in mocha. Some of our packages already have things like this, for example collector has common tests and then separate for node and browser.